### PR TITLE
Tries to fix plasma wizards

### PR DESCRIPTION
**Pull request by Aurorablade**
*the suit spawn code for this stuff is odd in the order it happens and i probaly get it wrong*

:cl: Code Fethas, Sprites Skittles
rscadd:There is now a plasmaman wizard suit so they do not spontaously explode into fire on spawn.
/:cl:

### CODE REVIEW

**Comment by Dinarzad** [comment]
That is the coolest lookin' shit.

File: code/modules/mob/living/carbon/human/species/plasmaman.dm [hunk 1]
```diff
@@ -148,8 +148,13 @@
 		if("Mime")
 			suit=/obj/item/clothing/suit/space/eva/plasmaman/mime
 			helm=/obj/item/clothing/head/helmet/space/eva/plasmaman/mime
-	H.equip_or_collect(new suit(H), slot_wear_suit)
-	H.equip_or_collect(new helm(H), slot_head)
+
+	if(H.mind.special_role == SPECIAL_ROLE_WIZARD || !H.mind.special_role == SPECIAL_ROLE_WIZARD_APPRENTICE)
```
Comment (Citinited) [inline code/modules/mob/living/carbon/human/species/plasmaman.dm]: IIRC, I think you need parentheses on the second bit of this i.e. ` !(H.mind.special_role == SPECIAL_ROLE_WIZARD_APPRENTICE))`.

**Comment by taukausanake** [comment]
It almost looks too cool. Like Quarterwizard cool. I like it thought but maybe it should be toned down a bit

File: code/modules/mob/living/carbon/human/species/plasmaman.dm (hunk 1 above)
Comment (RyanSmake) [reply code/modules/mob/living/carbon/human/species/plasmaman.dm]: *cough* `!=` *cough*

Comment (RyanSmake) [reply code/modules/mob/living/carbon/human/species/plasmaman.dm]: Or if you want to be very very very annoying use `<>`

**Comment by granodd** [comment]
He looks like he's about to Ice Barrage a fool. I wonder if you can get it to come in an alternate color of black/red.

**Comment by davidchan13** [comment]
Wouldn't it be easier and more lore friendly to just have all wizards forced human? Last I remember Vox had similiar issues with N2 tanks not spawning for them.

**Comment by taukausanake** [comment]
I wouldn't imagine the Wizard Federation to be racist to other species but I think it would be better for them to just be humans

**Comment by Aurorablade** [comment]
Code > lore.
Anywa yeah theres a red and black version but i dunno if i want to go into that much detail having it alternate. and i will see if i can fix the vox O2 tank thing.

**Pull request merged**